cookiecutter.json

```diff
@@ -13,14 +13,15 @@
     "docker_base": [
         "alpine:3.7",
         "alpine:3.8",
+        "centos:7",
+        "custom:special-image",
```

Comment:
what is special about this one?

Reply:
it is only to ease the creation of custom images (such as based on images not available in this list, e.g. nvidia-stuff or matlab...)

Reply:
Would `custom:image-template` be a better name?